detox/src/devices/drivers/android/genycloud/services/GenyInstanceLifecycleService.js

```diff
@@ -0,0 +1,25 @@
+const Instance = require('./dto/GenyInstance');
+
+class GenyInstanceLifecycleService {
+  constructor(genyCloudExec, instanceNaming) {
+    this.genyCloudExec = genyCloudExec;
+    this.instanceNaming = instanceNaming;
+  }
+
+  async createInstance(recipeUUID) {
+    const result = await this.genyCloudExec.startInstance(recipeUUID, this.instanceNaming.generateName());
+    return new Instance(result.instance);
+  }
+
+  async adbConnectInstance(instanceUUID) {
+    const result = (await this.genyCloudExec.adbConnect(instanceUUID));
+    return new Instance(result.instance);
+  }
+
+  async deleteInstance(instanceUUID) {
```

Comment:
Where is this method used from (aside unit tests, of course)?

Reply:
'Introduced it too soon. It is used now following recent tear-down code.